### PR TITLE
Mobile menu with JavaScript Pull request

### PR DESCRIPTION
**Pull request by HERMON-1995**
-  This project is all about building the mobile menu option using JavaScript DOM manipulation. Here is some of my changes:
                                           

                      1. Added mobile navigation bars in HTML file.
                      2. Added new JS file to implement the responsiveness of an event.
                      3. Added eslintrc.json file to check and fix errors in JavaScript code.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the javascript linters also setup, for this project both the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters should be configured
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by huseyinbiyik** [comment]
Hi @HERMON-1995 and @Ashnaali3255,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that is only for Eslint and Stylint linters. You can set up linters with [HTML&CSS&JavaScript](https://github.com/microverseinc/linters-config/tree/master/html-css-jsl) altogether.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -11,15 +11,22 @@
   <div>
     <header class="header">
      <h1 class="logo">Welcome</h1>
```
Comment (huseyinbiyik) [inline index.html:13]: Good job with the functionality but let's stick to the design provided and remove 'Welcome' text on our mobile nav.
Screenshot 2023-01-30 at 11 02 38 PM

File: style.css [hunk 2]
```diff
@@ -35,12 +36,66 @@ body {
...
+  display: block;
+  margin-left: 28px;
+  margin-bottom: 40px;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #3a4a42;
 }
 
 .nav-menu {
```
Comment (huseyinbiyik) [inline style.css:101]: Links are going where they need to go 👌 But let's follow the design and rectify the font-family of them.
Screenshot 2023-01-30 at 11 03 31 PM

Screenshot 2023-01-30 at 11 03 08 PM

File: index.html [hunk 3]
```diff
@@ -11,15 +11,22 @@
...
-          <li><a href="#contact">Contact</a></li>
-        </ul>
+     <nav class="site-nav">
+       <button class="hamburger"><div class="bar" title="bars"></div></button>
+    <ul class="nav-menu">
+       <li><a href="#portfolio">Portfolio</a></li>
+       <li><a href="#about">About</a></li>
+       <li><a href="#contact">Contact</a></li>
+     </ul>
      </nav>
```
Comment (huseyinbiyik) [inline index.html:21]: Kindly fix the bottom of the mobile navigation like I marked below. It is not a full screen. There is a gap under the nav and it has a border-radius for both sides.
Screenshot 2023-01-30 at 10 59 35 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- EsLint set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] You can use a fixed header like on the Figma template [here](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48:988&scaling=min-zoom&page-id=1:468) to increase the accessibility of your web page.
-  Please make sure your commit messages are meaningful and descriptive for your next commits. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by osamaashraf6 — APPROVED** [review]
Hi @HERMON-1995 , 
# STATUS: APPROVED! 🥇🟢🟢💯 


Your project is complete!  There is nothing else to say other than... it's time to merge it : ship it: 
Congratulations! 🎉 

## Highlights💯🎈🪄✔️

- [x] Good commit messages.✔️
- [x] Descriptive READme file in a professional way.✔️
- [x] No linter errors. ✔️
- [x] Good usage in correcting GitHub flow.✔️
- [x] Nice design.✔️

## Optional Suggestions

- [ ] Kindly try to fix the horizontal scrolling of the body to let the portfolio look nice for the visitors😊


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**